### PR TITLE
Added options to build ODBC driver with iODBC or unixODBC

### PR DESCRIPTION
**Pull request by aseidlitz**

### CODE REVIEW

File: Library/Formula/mdbtools.rb [hunk 1]
```diff
@@ -5,12 +5,16 @@ class Mdbtools < Formula
   sha256 "dcf310dc7b07e7ad2f9f6be16047dc81312cfe1ab1bd94d0fa739c8059af0b16"
 
   option "with-man-pages", "Build manual pages"
+  option "with-iodbc", "Build with iODBC"
+  option "with-unixodbc", "Build with unixODBC"
```
Comment (mikemcquaid) [inline Library/Formula/mdbtools.rb]: You can remove these options.

File: Library/Formula/mdbtools.rb [hunk 2]
```diff
@@ -5,12 +5,16 @@ class Mdbtools < Formula
...
 
   depends_on "pkg-config" => :build
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "txt2man" => :build if build.with? "man-pages"
+  depends_on "libiodbc" => :optional if build.with? "iodbc"
```
Comment (mikemcquaid) [inline Library/Formula/mdbtools.rb]: Just use `:optional` here and remove the `build.with?` and it'll generate an option called `--with-libiodbc`

File: Library/Formula/mdbtools.rb [hunk 3]
```diff
@@ -18,7 +22,11 @@ def install
     ENV.deparallelize
 
     args = ["--prefix=#{prefix}"]
+    args << "--with-iodbc=#{Formula["libiodbc"].opt_prefix}" if build.with? "iodbc"
+    args << "--with-unixodbc=#{Formula["unixodbc"].opt_prefix}" if build.with? "unixodbc"
+    #args << "--with-unixodbc=#{HOMEBREW_PREFIX}" if build.with? "unixodbc"
     args << "--disable-man" if build.without? "man-pages"
+
```
Comment (mikemcquaid) [inline Library/Formula/mdbtools.rb]: Remove this blank line.

File: Library/Formula/mdbtools.rb [hunk 4]
```diff
@@ -18,7 +22,11 @@ def install
     ENV.deparallelize
 
     args = ["--prefix=#{prefix}"]
+    args << "--with-iodbc=#{Formula["libiodbc"].opt_prefix}" if build.with? "iodbc"
+    args << "--with-unixodbc=#{Formula["unixodbc"].opt_prefix}" if build.with? "unixodbc"
+    #args << "--with-unixodbc=#{HOMEBREW_PREFIX}" if build.with? "unixodbc"
```
Comment (mikemcquaid) [inline Library/Formula/mdbtools.rb]: remove this line.

**Comment by bfontaine** [comment]
Ping re Mike’s comments.

**Comment by bfontaine** [comment]
Closing due to no response. Feel free to open another PR addressing the issues above. Thanks!

**Pull request closed without merging**